rainforest/rfml.go

```diff
@@ -124,6 +126,26 @@ func (r *RFMLReader) ReadAll() (*RFTest, error) {
 						return parsedRFTest, &parseError{lineNumStr, "Redirect value must be a valid boolean"}
 					}
 					currStepRedirect = redirect
+				case "feature_id":
+					if value == "" {
+						// If the value is empty, delete the feature
```

Comment:
To clarify: this won't get triggered if `feature_id` is blank?

Reply:
We won't reach this case if the file does not say "# feature_id:" in it.